conf/supervisord-dev.conf

```diff
@@ -1,8 +1,14 @@
 [supervisord]
 nodaemon = true
-environment=PYTHONUNBUFFERED="1"
 silent = true
 
+[program:app]
+command=python3 -m h_periodic
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
```

Comment:
This is another problem with the `supervisord[-dev].conf` templates in the cookiecutter: they always insert this default `app` program but it's not always wanted. `h-periodic` actually doesn't want to have this program.